middleware/cache/cache_test.go

```diff
@@ -107,6 +108,110 @@ func Test_Cache(t *testing.T) {
 	utils.AssertEqual(t, cachedBody, body)
 }
 
+func Test_Cache_WithCacheControlNoCacheRequestHeader(t *testing.T) {
```

Comment:
added Test_Cache_WithCacheControlNoCacheRequestHeader and Test_Cache_WithETagAndCacheControlNoCacheRequestHeader